farm/data_handler/processor.py

```diff
@@ -1744,10 +1745,13 @@ def _apply_tokenization(dictionary, tokenizer):
             external_id = question["id"]
             question_text = question["question"]
             for answer in question["answers"]:
-                if answer["text"] == "":
-                    answer_type = "no_answer"
+                if 'answer_type' in answer.keys() and answer['answer_type'] in ['yes', 'no', 'span', 'no_answer']:
```

Comment:
I am unhappy about having the answer types hard coded here as a list. We should somehow link the already defined answer_type_list attribute of NaturalQuestionsProcessor.

Reply:
Can we finish this before the next release?